src/components/app/QuickActions.js

```diff
@@ -262,18 +248,37 @@ export class QuickActions extends Component {
         childView,
         parentView
       )
-        .then(response => {
+        .then(result => {
+          const [respRel, resp] = result;
+
           if (this.mounted) {
+            const currentActions =
+              resp && resp.data ? resp.data.actions : respRel.data.actions;
+            const relatedActions =
+              resp && resp.data ? respRel.data.actions : null;
+
+            console.log('response: ', relatedRef, currentActions, relatedActions, resp, respRel);
+
+            if (relatedRef && relatedRef.current && parentView.viewId && relatedActions) {
+              this.props.relatedRef.current.handleUpdateQuickActions(relatedActions);
+            }
+
+            if (relatedRef && relatedRef.current && childView.viewId && relatedActions) {
+              this.props.relatedRef.current.handleUpdateQuickActions(relatedActions);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['relatedRef.current.handleUpdateQuickActions' is missing in props validation (react/prop-types)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4613375)